src/main/java/sorald/processor/ToStringReturningNullProcessor.java

```diff
@@ -0,0 +1,30 @@
+package sorald.processor;
+
+import sorald.Constants;
+import sorald.annotations.IncompleteProcessor;
+import sorald.annotations.ProcessorAnnotation;
+import spoon.reflect.code.CtReturn;
+import spoon.reflect.declaration.CtMethod;
+import spoon.reflect.visitor.filter.TypeFilter;
+
+@IncompleteProcessor(description = "does not fix null returning clone()")
+@ProcessorAnnotation(
+        key = 2225,
+        description = "\"toString()\" and \"clone()\" methods should not return null")
+public class ToStringReturningNullProcessor extends SoraldAbstractProcessor<CtReturn<?>> {
+
+    @Override
+    protected boolean canRepairInternal(CtReturn<?> candidate) {
+        CtMethod parentMethod = candidate.getParent(new TypeFilter<>(CtMethod.class));
+        if (parentMethod.getSignature().equals(Constants.TOSTRING_METHOD_NAME + "()")
+                && candidate.getReturnedExpression().toString().equals("null")) {
+            return true;
+        }
+        return false;
```

Comment:
This can be simplified. In general, anything on this form:

```java
if (something) {
    return true;
}
return false;
```
can be simplified like so:

```java
return something;
```
So, it would make it something like this:

```suggestion
        return parentMethod.getSignature().equals(Constants.TOSTRING_METHOD_NAME + "()")
                && candidate.getReturnedExpression().toString().equals("null");
```